src/stores/DiscoverQueryParamsStore.ts

```diff
@@ -34,7 +34,7 @@ export const defaultInitState: DiscoverQueryParamsStoreState = {
     creatorSlug: undefined,
     search: undefined,
     filterTag: undefined,
-    sortTag: ComicSortTag.Title,
```

Comment:
why?

Reply:
The output is the same on Discover screen, but it removes filter count (if we put some tags here, excluding SorOrder ones, user sees one filter preselected on Discover page load)